normandy/recipes/management/commands/initial_data.py

```diff
@@ -41,3 +42,16 @@ def add_countries(self):
         for code, name in countries:
             Country.objects.update_or_create(code=code, defaults={"name": name})
         self.stdout.write("Done")
+
+    def add_windows_versions(self):
+        self.stdout.write("Adding Windows Versions...", ending="")
+        versions = {
+            "6.1": "Windows 7",
+            "6.2": "Windows 8",
+            "6.3": "Windows 8.1",
+            "10.0": "Windows 10.0",
+        }
+
+        for slug, name in versions.items():
+            WindowsVersion.objects.update_or_create(slug=slug, defaults={"name": name})
```

Comment:
As a stylistic note, not a functional one: I'd use a tuple here instead of a key/value pair, since this isn't really being used as a lookup:

```suggestion
        versions = [
            ("6.1", "Windows 7"),
            ("6.2", "Windows 8"),
            ("6.3", "Windows 8.1"),
            ("10.0", "Windows 10"),
        ]

        for slug, name in versions:
            WindowsVersion.objects.update_or_create(slug=slug, defaults={"name": name})
```